master/internal/rm/agentrm/agent_resource_manager.go

```diff
@@ -579,8 +581,9 @@ func (a *ResourceManager) ValidateResourcePool(name rm.ResourcePoolName) error {
 
 // DefaultNamespace is not supported.
 func (a *ResourceManager) DefaultNamespace(string) (*string, error) {
-	return nil, fmt.Errorf("resource manager type AgentRM does not have a default namespace: %w",
-		rmerrors.ErrNotSupported)
+	return nil, status.Error(codes.NotFound,
```

Comment:
probably we wanna use `NotFoundErrs()`? other places use the func

https://github.com/determined-ai/determined/blob/c97f3741e58220fec6a1e6fc603f1ddd97a66ade/master/internal/api/error.go#L79-L85

Reply:
We don't have an id to pass into it. I think my code is essentially doing the same thing

Reply:
do we need to change any of the other RM errors?